### ISSUE TITLE
`okhttp` RequestBody.create(mediaType,body) -> RequestBody.create(body,mediaType)

### ISSUE THREAD

**Issue by yeikel** [openrewrite/rewrite-migrate-java#102]
RequestBody.create(mediaType,body) -> RequestBody.create(body,mediaType)

See https://github.com/square/okhttp/issues/6655

**Comment by tkvangorder** [comment]
Yeah, this is a very straightforward recipe. We don't really have a set place to put a libary-specific recipe for OkHttp, so it makes sense to host it here (unless Square/Block wants to host this recipe).

**Comment by tkvangorder** [comment]
@escardin Do you know anyone on the OkHttp team? 😄

**Comment by swankjesse** [comment]
Hi! I’m on the OkHttp team.

**Comment by tkvangorder** [comment]
Hello @swankjesse !

We were wondering if your team is interested in hosting the recipe (and others) for helping migrate your users off of deprecated API or helping automate a migration when you jump to a new major version of your libraries?

We can certainly host these one-off recipes here but then there is an issue of discoverability. Happy to discuss the capabilities of the framework with you.

**Comment by timtebeek** [comment]
@swankjesse We've come a long way since 2022 with our support for Kotlin and community maintained migration recipes. Would you be interested in collaborating on a `rewrite-okhttp` recipe module?

We've recently (re)started such modules for micronaut,  jenkins, cucumber-jvm and testcontainers among others, where the library/framework authors are involved with recipe development, and we facilitate that by supporting, documenting and distributing those recipes in tandem with new OpenRewrite releases. That way it should be easy to bring your users along to newer versions, and focus your efforts only on the latest version, and getting users there.

From this issue and [the changelog](https://square.github.io/okhttp/changelogs/changelog/) it seems it should be easy to support at least part of the migration, especially as our Kotlin support includes running recipes for Java. Most changes look to be possible with declarative recipes only, so we'd mostly only need the repository set up with perhaps some tests for verify the changes.

**Comment by swankjesse** [comment]
Awesome.
